packages/fluentui/docs/src/views/Accessibility.tsx

```diff
@@ -383,6 +383,37 @@ export default () => (
       `}
     />
 
+    <Header as="h3" content="Useful examples of labelling" />
+    <Header as="h4" content="Aria-labelledby refers to itself " />
+    <p>
+      In following example we want to get narrated from screen reader "delete Active members" button. Solution is
```

Comment:
There are cases, when `aria-label` and `aria-labelledby` when you need to add names of other elements in the accessible name of an element. In the example below, the button has accessible name composed of its own name (delete) and header name (Active members). Solution is to refer to the element itself by its `id` in the `aria-labelledby` attribute. Screen reader
      computes the accessible name in the order which is provided in `aria-labelledby`. The strategy was taken from the
      {link(' Accessible Name and Description Computation 1.1', 'https://www.w3.org/TR/accname-1.1/#terminology')} page.

Reply:
applied changes